presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -672,20 +699,21 @@ public Void visitRemoteSource(RemoteSourceNode node, Void context)
         {
             for (PlanFragmentId planFragmentId : node.getSourceFragmentIds()) {
                 PlanFragment target = fragmentsById.get(planFragmentId);
-                printEdge(node, target.getRoot());
+                printEdge(node, target.getRoot(), NO_LABEL);
             }
 
             return null;
         }
 
-        private void printEdge(PlanNode from, PlanNode to)
+        private void printEdge(PlanNode from, PlanNode to, String label)
```

Comment:
you can try with  overloading `printEdge` to simply the call so no need to specify `NO_LABEL` to the mainline logic like `visitPlan` or `visitRemoteSource`

Reply:
I was waiting for one more `printEdge` call (currently 2) to use NO_LABEL to add such an overload :)